zerver/lib/bugdown/__init__.py

```diff
@@ -1173,6 +1174,15 @@ def run(self, lines):
                 if re.search(match_re, content):
                     current_message.alert_words.add(word)
 
+            for command in slash_commands:
```

Comment:
Either approaches would work for me. But I think I would slightly prefer current approach more, because this is our initial version, and it is possible that we may start using alert words for users instead of short name as slash commands in future.